guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -59,6 +59,26 @@ euid = __ID_of_Apache_User__
 # chown root.{apache-user} /etc/httpd/conf/http.keytab
```

Comment:
FYI, since I realize this is outside the scope of your PR. [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#commands-in-code-blocks) says to use only one command per code block.

Reply:
I actually didn't know that, thank you. But yes, let's not worry about it now, I'm pretty sure we'll need to revisit this procedure anyway and things might look quite different in the end.